config/initializers/devise.rb

```diff
@@ -14,7 +14,7 @@
   # confirmation, reset password and unlock tokens in the database.
   # Devise will use the `secret_key_base` as its `secret_key`
   # by default. You can change it below and use your own secret key.
-  config.secret_key = Rails.application.secrets.secret_key_base
+  config.secret_key = Rails.application.secret_key_base
```

Comment:
just double checking the existing value was `nil`, I'm worried that we might be changing it now (correctly) and will cause invalidation everywhere. Just confirming, let's hold off merging pelase